python/tank/commands/push_pc.py

```diff
@@ -284,7 +284,7 @@ def _run(self, log, target_id, use_symlink=False):
             try:
                 # copy everything!
                 log.debug("Copying %s -> %s" % (source_path, target_tmp_path))
-                filesystem.copy_folder(source_path, target_tmp_path)
+                filesystem.copy_folder(source_path, target_tmp_path, skip_list=[])
```

Comment:
this is the fix for copying the `.git` folder while doing `tank push_configuration` :)